libs/transfer-journey/shell/src/lib/make-transfer-journey-store-guard.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRoute, CanActivate, Router } from '@angular/router';
+
+@Injectable()
+export class MakeTransferJourneyStoreGuard implements CanActivate {
+  canActivate() {
+    const state = this.router.getCurrentNavigation()?.extras?.state;
+    if (!state || !state['transfer']) {
+      return this.router.parseUrl('../make-transfer');
+    }
+    return true;
+  }
+  constructor(
+    private readonly router: Router,
+    private readonly route: ActivatedRoute
```

Comment:
We are not using `route` anywhere so this could be removed.
Does it make sense to do this in this PR, it's just for updating journeys to follow DDD?

Reply:
Lets not review the code for this PR . Though, I will remove this for now.